search.json

```diff
@@ -35,4 +35,14 @@ layout:
 "date":"{{ post.date }}"
 } {% unless forloop.last %},{% endunless %}
 {% endfor %}
-]
+{% for jobs in site.jobs %}
+{
+"type":"Jobs",
+"title":"{{ jobs.title | escape }}",
+"summary":"{{ jobs.body | escape | truncatewords: 40 | strip | prepend: ' ' }}",
+"subtitle":"{{ jobs['Place of Work'] | join: ', ' }}",
```

Comment:
This text block will need to be stripped and escaped or there's a chance the resulting json will be invalid.